lib/buildkit/header_link_parser.rb

```diff
@@ -0,0 +1,26 @@
+module HeaderLinkParser
```

Comment:
Should this be namespaced under the Buildkit module like all the other constants?  The file path indicates that it should be namespaced.  On the other hand, the link header format isn't buildkite specific (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Link).

Reply:
Absolutely, that's an oversight.